go/libkb/constants.go

```diff
@@ -55,7 +55,10 @@ const (
 	KeybaseSaltpackBrand = "KEYBASE"
 )
 
-var UserAgent = "Keybase/" + Version + " (" + runtime.Version() + " on " + runtime.GOOS + ")"
+// Right now reddit is the only site that seems to have any requirements for
+// our User-Agent string. (See https://github.com/reddit/reddit/wiki/API.)If
+// something else comes up, we'll want to make this more configurable.
+var UserAgent = runtime.GOOS + ":" + "Keybase CLI (" + runtime.Version() + "):" + Version + " (by /u/oconnor663)"
```

Comment:
To avoid too much nasty plumbing, you can ask if the host is `reddit.com` in `fixHeaders`...